### ISSUE TITLE
[TESTING]: Icons SHOULD have unit tests

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#10431]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Our icons should have unit tests to evaluate props and accessibility. This will also encourage new tests to be written if the component(s) change in the future.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Unit tests are written for the Icons (Base, Help, Search, User)
- [ ] Icons Help, Search, User should have 0 axe violations

## Environment

* https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/iconbase/
* https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/iconhelp/
* https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/iconsearch/
* https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/iconuser/

**Comment by 1Copenut** [comment]
Issue moved to [department-of-veterans-affairs/vets-design-system-documentation #222](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/222) via [**ZenHub**](https://www.zenhub.com/)
